lib/runnerman.js

```diff
@@ -1,18 +1,19 @@
 const { Collection, Event, Script } = require('postman-collection');
 const { run: newman } = require('newman');
+const { format } = require('util');
+const { v4: uuidv4 } = require('uuid');
 const { filterCollection } = require('./util/filterCollection');
 const { generateGlobals } = require('./util/generateGlobal');
-const { format } = require('util');
 const { createGlobalVariable } = require('./util/createGlobalVariable');
-const { v4: uuidv4 } = require('uuid');
 
-module.exports = async function (suite, collectionDefinition, environment, iterations) {
+
+module.exports = async function runnerman(suite, collectionDefinition, environment, iterationCount, reporters) {
     let guid = null;
 
-    const promises = new Promise((resolve, reject) => {
-        const name = suite.name;
+    let consoleColor = '\x1b[32m';
 
-        const suiteObj = suite.obj;
+    const promises = new Promise((resolve, reject) => {
```

Comment:
Async func é Promise. Aqui poderia ser async func tbm

Reply:
So, I really need `Promise` here, because `Newman.run` is a `EventEmitter` and I really wish resolve this promise after event `done`.